src/Microsoft.Azure.WebJobs.Logging.ApplicationInsights/FilteringTelemetryProcessor.cs

```diff
@@ -41,6 +42,11 @@ private bool IsEnabled(ITelemetry item)
                 {
                     // If no category is specified, it will be filtered by the default filter
                     categoryName = string.Empty;
+
+                    if (telemetry is OperationTelemetry)
+                    {
+                        enabled = false;
+                    }
```

Comment:
This is where the "magic" happens, right? We'll only log OperationTelemetry if it has a category attached -- and that will only happen when wrapped in our scope.

It may be worth putting a comment here for others to briefly explain how this works.

Reply:
Exactly, this is the place that filters unwanted calls. I'll add a comment, good point.